### PR TITLE
New LED mount, exhaust filter housing and panel clip

### PR DESCRIPTION
**Pull request by 42bios**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
a new LED mount, a new exhaust filter housing and a panel clip with cable path
#### How was it tested? 
printed and used on my voron
#### Screenshots (if appropriate)
in folders

### CODE REVIEW

**Comment by 42bios** [comment]
Added corner panel clip with cable path

File: printer_mods/42bios/corner_panel_clip_cable/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Corner panel clip with a internal cable path, so you can route the cables around the panel and corner.<br>
```
Comment (KiloQubit) [inline printer_mods/42bios/corner_panel_clip_cable/README.md:1]: Replace `a internal` with `an internal`.

File: printer_mods/42bios/corner_panel_clip_cable/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+I use them to mount 6mm floatglas as top panel with 3mm foamtape<br>
+<br>
+corner_panel_clip_top_4mm.stl is 4mm and corner_panel_clip_bottom_3_5mm.stl is 3.5mm hight, <br>
```
Comment (KiloQubit) [inline printer_mods/42bios/corner_panel_clip_cable/README.md:4]: Replace `hight` with `height`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!  Please see requested changes below.

- `exhaust_filter_housing_side/exhaust_filter_housing_side.textClipping` - remove this file, and add the missing STL for the exhaust housing
- `LED_mount/AngryLED_{l,r}_v1.stl` - add a chamfer under the unsupported overhang in both files
Screen Shot 2021-05-01 at 11 55 34

**Comment by 42bios** [comment]
@KiloQubit thanks for the hint! everything should be right now

**Comment by KiloQubit** [comment]
`LED_mount/AngryLED_r_v1.stl` looks completely broken now
Screen Shot 2021-05-02 at 11 31 11

Same broken problem with `LED_mount/AngryLED_small_2_v1.stl`.

Also, the large AngryLED STLs are marked l/r (left/right I assume), but the small ones are 1/2 - maybe both should use the same convention.

**Comment by 42bios** [comment]
updated and renamed it :)

**Comment by KiloQubit** [comment]
Thanks!  One more thing I missed earlier - the fillets on the bottom of all the AngryLED STLs result in an overhang steeper than 45 degrees.  Can you replace them with chamfers, or a combination of chamfer/fillet like on your panel clips?

Screen Shot 2021-05-07 at 08 18 46

**Comment by 42bios** [comment]
I give my best and try it :D

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**